drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -88,4 +88,34 @@ config ETH_STM32_CARRIER_CHECK_RX_IDLE_TIMEOUT_MS
 	  Set the RX idle timeout period in milliseconds after which the
 	  PHY's carrier status is re-evaluated.
 
+if SOC_SERIES_STM32F7X || SOC_SERIES_STM32H7X
+
+config PTP_CLOCK_STM32_HAL
+    bool "STM32 HAL PTP clock driver support"
+    default y
+    depends on PTP_CLOCK || NET_L2_PTP
+    help
+      Enable STM32 PTP clock support.
+
+if PTP_CLOCK_STM32_HAL
+
+config ETH_STM32_HAL_PTP_CLOCK_SRC_HZ
+    int "Frequency of the clock source for the PTP timer"
+    default 50000000
```

Comment:
Is this the default? I thought it was 100 MHz

Reply:
It can be whatever we want really - I just stuck with the same one they used for the other board.